### PR TITLE
[DOCS] CanvasItem(Material)/Layer, ClassDB, and CollisionObject(2D)

### PR DESCRIPTION
**Pull request by willnationsdev**

### CODE REVIEW

File: doc/classes/CollisionObject.xml [hunk 1]
```diff
@@ -181,8 +183,10 @@
 	</methods>
 	<members>
 		<member name="input_capture_on_drag" type="bool" setter="set_capture_input_on_drag" getter="get_capture_input_on_drag">
+			If [code]true[/code] will continue to receive input events as a mouse is dragged across the [code]CollisionObject[/code]'s shapes. Default value: [code]false[/code].
```
Comment (cbscribe) [inline doc/classes/CollisionObject.xml]: Missing subject. If true, what will continue to receive input events?

**Review by cbscribe — CHANGES_REQUESTED** [review]

File: doc/classes/CanvasItem.xml [hunk 2]
```diff
@@ -438,20 +448,28 @@
...
+			The material applied to textures on this [code]CanvasItem[/code]. Default value: [code]null[/code].
 		</member>
 		<member name="modulate" type="Color" setter="set_modulate" getter="get_modulate">
+			The color applied to textures on this [code]CanvasItem[/code]. Default value: [code]Color(1, 1, 1, 1)[/code] (opaque "white").
 		</member>
 		<member name="self_modulate" type="Color" setter="set_self_modulate" getter="get_self_modulate">
+			The color applied to textures on this [code]CanvasItem[/code]. This is not inherited by children [code]CanvasItem[/code]s. Default value: [code]Color(1, 1, 1, 1)[/code] (opaque "white")..
 		</member>
 		<member name="show_behind_parent" type="bool" setter="set_draw_behind_parent" getter="is_draw_behind_parent_enabled">
+			If [code]true[/code] draws the parent [code]CanvasItem[/code] over this one. Default value: [code]false[/code].
```
[the remark this answers is not in the document]
Comment (willnationsdev) [reply doc/classes/CanvasItem.xml]: Wouldn't that be passive though?

[the remark this answers is not in the document]
Comment (willnationsdev) [reply doc/classes/CanvasItem.xml]: perhaps do "renders" or "appears" insteadn of "is drawn"?

[the remark this answers is not in the document]
Comment (willnationsdev) [reply doc/classes/CanvasItem.xml]: But "is drawn" is just simpler, easier to read, and more intuitive...I'll just do that.

File: doc/classes/CanvasItem.xml [hunk 3]
```diff
@@ -438,20 +448,28 @@
...
+			The material applied to textures on this [code]CanvasItem[/code]. Default value: [code]null[/code].
 		</member>
 		<member name="modulate" type="Color" setter="set_modulate" getter="get_modulate">
+			The color applied to textures on this [code]CanvasItem[/code]. Default value: [code]Color(1, 1, 1, 1)[/code] (opaque "white").
 		</member>
 		<member name="self_modulate" type="Color" setter="set_self_modulate" getter="get_self_modulate">
+			The color applied to textures on this [code]CanvasItem[/code]. This is not inherited by children [code]CanvasItem[/code]s. Default value: [code]Color(1, 1, 1, 1)[/code] (opaque "white")..
 		</member>
 		<member name="show_behind_parent" type="bool" setter="set_draw_behind_parent" getter="is_draw_behind_parent_enabled">
+			If [code]true[/code] the parent [code]CanvasItem[/code] is drawn over this one. Default value: [code]false[/code].
```
Comment (cbscribe) [inline doc/classes/CanvasItem.xml]: Yeah, how about "If [code]true[/code] the object draws behind its parent."

**Review by cbscribe — APPROVED** [review]

**Pull request merged**